Tests/PointFreeTests/SubscribeConfirmationTests.swift

```diff
@@ -237,6 +344,42 @@ class SubscriptionConfirmationTests: TestCase {
     }
     #endif
   }
+
+  func testTeam_LoggedIn_RemoveOwnerFromTeam() {
+    update(
+      &Current,
+      \.database.fetchUserById .~ const(pure(.mock |> \.gitHubUserId .~ -1)),
+      \.database.fetchSubscriptionById .~ const(pure(nil)),
+      \.database.fetchSubscriptionByOwnerId .~ const(pure(nil))
+    )
+
+    let conn = connection(
+      from: request(
+        to: .subscribeConfirmation(
+          lane: .team,
+          billing: nil,
+          isOwnerTakingSeat: nil,
+          teammates: nil
+        ),
+        session: .loggedIn
+      )
+    )
+    let result = conn |> siteMiddleware
+
+    #if !os(Linux)
+    if self.isScreenshotTestingAvailable {
+      let webView = WKWebView(frame: .init(x: 0, y: 0, width: 1100, height: 1600))
+      let html = String(decoding: result.perform().data, as: UTF8.self)
+      webView.loadHTMLString(html, baseURL: nil)
+
+      assertSnapshot(
+        matching: webView,
+        as: .image(afterEvaluatingJavascript: "document.getElementById('remove-yourself-button').click()"),
```

Comment:
this tests what happens when a user actually removes themselves from the subscription.